op-verifier/contracts/interfaces/IDisputeGame.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This needs to include some information on where it's sourced from, as well as copyright holders.

Reply:
I have build another NPM package and use foundry flatten on optimism interfaces. It should include information on where it's sourced from as per foundry standard.

https://github.com/Opti-domains/dispute-game-lookup/blob/main/npm/contracts/interfaces/optimism/IDisputeGameFactory.sol